service/src/main/java/de/adorsys/psd2/sandbox/xs2a/service/ais/AccountSpiImpl.java

```diff
@@ -224,4 +225,8 @@ public AccountSpiImpl(TestDataService testDataService, TestDataMapper testDataMa
     return testDataService
         .getTransactions(psu.get().getPsuId(), accountId);
   }
+
+  private boolean dateIsInDateRange(LocalDate date, LocalDate from, LocalDate to) {
+    return date.compareTo(from) > 0 && date.compareTo(to) < 0;
```

Comment:
`LocalDate#before/after` is easier to read.

Reply:
Done